### DISCUSSION TITLE
Hourly reminder bot in discord

### DISCUSSION THREAD

**Discussion by rodrigoc211** [discordjs/discord.js#6705] [Q&A] [answered]
Hii
Is it possible to make a hourly reminder bot to my server discord?
I'm struggling to find out.
Thank you!!

**Comment by KinectTheUnknown** [comment, marked as answer]
You can use a cron package like [node-cron](https://www.npmjs.com/package/node-cron), but that is outside of the scope of this library.

**Reply by rodrigoc211** [reply]
Okk,
I will try, thank you!
